lib/plugins/blocks.js

```diff
@@ -135,13 +135,12 @@ function inject (bot, { version }) {
 
   bot.findBlocks = (options) => {
     const matcher = getMatchingFunction(options.matching)
-    const point = options.point || bot.entity.position
-    let start = point.floored()
-    const maxDistance = Math.ceil((options.maxDistance || 16) / 16)
+    const point = (options.point || bot.entity.position).floored()
```

Comment:
Are you sure about this part ? It seems to be undoing the previous PR

Reply:
yeah I'm sure, it's to correct a small bias in the distance. The previous pr was to use the point instead of the section in which the point was. This fix floor the point to avoid biasing toward one side of the block.